README.md

```diff
@@ -138,6 +138,12 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 - [typioca](https://github.com/bloznelis/typioca) - A typing test. (_built with Bubble Tea_)
 - [wordle](https://github.com/sa-/wordle-tui) - A wordle game for your terminal! (_built with Bubble Tea_)
 
+### Messaging and Communication
```

Comment:
I wonder if we need to make a new category 🤔

Reply:
I didn't find any matching category. Can you suggest one.

Reply:
I guess a new category will work. Thanks for the PR!